droopler.install

```diff
@@ -1019,6 +1019,7 @@ function droopler_update_8106() {
     ],
     'profile/droopler' => [
       'filter.format.full_html.yml' => '',
+      'editor.editor.full_html.yml' => '',
```

Comment:
Hm, to chyba nie zadziała przy update, bo ten config istniał wcześniej, więc powinniśmy mieć hasha